php/class-plugin.php

```diff
@@ -37,6 +37,7 @@ public function __construct() {
 	function init() {
 		add_action( 'wp_default_scripts', array( $this, 'register_scripts' ), 11 );
 		add_action( 'wp_default_styles', array( $this, 'register_styles' ), 11 );
+		add_action( 'admin_init', array( $this, 'add_caps' ) );
```

Comment:
This could be simplified to
```PHP
/**
 * Add the customize_publish capability to users who can edit_theme_options by default.
 *
 * @param array $allcaps An array of all the user's capabilities.
 */
function filter_user_has_cap( $allcaps ) {
    if ( ! empty( $allcaps['edit_theme_options'] ) ) {
        $allcaps['customize_publish'] = true;
    }
    return $allcaps;
}
```